com.unity.sg2/Editor/GraphUI/DataModel/Searcher/ShaderGraphSearcherDatabaseProvider.cs

```diff
@@ -39,6 +39,67 @@ public override IReadOnlyList<ItemLibraryDatabaseBase> GetVariableTypesDatabases
             return databases;
         }
 
+        public override IReadOnlyList<ItemLibraryDatabaseBase> GetGraphElementContainerDatabases(GraphModel graphModel, IGraphElementContainer container)
+        {
+            if (container is not GraphDataContextNodeModel contextNode || graphModel is not ShaderGraphModel sgModel)
+            {
+                return base.GetGraphElementContainerDatabases(graphModel, container);
+            }
+
+            var availableBlocks = new List<ItemLibraryItem>();
```

Comment:
I'd like this next section abstracted out and broken down a bit so it wasn't such a large block, want to make it clear that this is the bit for handling gathering context blocks